threads-api/src/threads-api.ts

```diff
@@ -166,6 +169,13 @@ export class ThreadsAPI {
     if (options?.deviceID) this.deviceID = options.deviceID;
     this.device = options?.device;
     this.userID = options?.userID;
+
+    if (options?.locale) {
+      this.locale = options.locale;
+    } else {
+      const detectedLocale: string = Intl.DateTimeFormat().resolvedOptions().locale;
+      this.locale = detectedLocale;
```

Comment:
can this value also be empty? (just asking, i dunno)

Reply:
I'm not sure, tbh its probably better to add a default value